src/d3a/models/myco_matcher/myco_external_matcher.py

```diff
@@ -51,28 +69,32 @@ def publish_offers_bids(self, message):
         # IDs of markets (Areas) the client is interested in
         filtered_areas_uuids = filters.get("markets")
         market_offers_bids_list_mapping = {}
-        for area_uuid, markets in self.area_uuid_markets_mapping.items():
+        for area_uuid, area_data in self.area_uuid_markets_mapping.items():
             if filtered_areas_uuids and area_uuid not in filtered_areas_uuids:
```

Comment:
```suggestion
            if area_uuid not in filtered_areas_uuids:
```

Reply:
This will break the functionality, if the client did not send filtered areas, `filtered_areas_uuids` will be None and then we can skip the condition
If I remove this check it will be if area_uuid in None -> Error

Reply:
Ohkay, I thought it would least be an empty list.